Features/Perpetuals/Sources/Views/AutocloseInputSection.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import SwiftUI
+import Components
+import Style
+import PrimitivesComponents
+
+struct AutocloseInputSection<Field: Hashable>: View {
+    @Binding var inputModel: InputValidationViewModel
+    let title: String
+    let placeholder: String
+    let estimatedPNLTitle: String
```

Comment:
```suggestion
    let pnlTitle: String
```

no need to use estimated